vnet/lib/vnet/endpoints/1.0/vnet_api.rb

```diff
@@ -115,6 +115,21 @@ def update_by_uuid(class_name, fill = {})
       respond_with(response.generate(updated_object))
     end
 
+    def update_by_uuid2(class_name, fill = {})
```

Comment:
Maybe add a big ugly deprecation message in the log to the old `update_by_uuid`?